### ISSUE TITLE
java.lang.ClassNotFoundException: kotlin.test.AssertionsKt PluginClassLoader

### ISSUE THREAD

**Issue by ghost** [patricklx/intellij-emberjs-experimental#8]
### Description

When opening an `.hbs` file there is a fatal error in WebStorm 2021.3.1.

Error:

```
com.intellij.diagnostic.PluginException: annotator: TemplateLintExternalAnnotator@42efcb10 (class TemplateLintExternalAnnotator) [Plugin: com.emberjs.experimental]
	at com.intellij.ide.plugins.PluginManagerCore.createPluginException(PluginManagerCore.java:270)
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:12)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:83)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.processError(ExternalToolPass.java:272)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.collectInformationWithProgress(ExternalToolPass.java:149)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:84)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:56)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$1(PassExecutorService.java:414)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1084)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:407)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:624)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:698)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:646)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:623)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:66)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:406)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:382)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:174)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:181)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:380)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:184)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
Caused by: java.lang.NoClassDefFoundError: kotlin/test/AssertionsKt
	at TemplateLintConfiguration.loadPrivateSettings(TemplateLintConfiguration.kt:20)
	at TemplateLintConfiguration.loadPrivateSettings(TemplateLintConfiguration.kt:11)
	at com.intellij.lang.javascript.linter.JSLinterConfiguration.getExtendedState(JSLinterConfiguration.java:60)
	at com.intellij.lang.javascript.linter.JSLinterConfiguration.getExtendedState(JSLinterConfiguration.java:45)
	at TemplateLintExternalAnnotator.collectInformation(TemplateLintExternalAnnotator.kt:62)
	at TemplateLintExternalAnnotator.collectInformation(TemplateLintExternalAnnotator.kt:12)
	at com.intellij.lang.javascript.linter.JSLinterWithInspectionExternalAnnotator.collectInformation(JSLinterWithInspectionExternalAnnotator.java:71)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.collectInformationWithProgress(ExternalToolPass.java:146)
	... 21 more
Caused by: java.lang.ClassNotFoundException: kotlin.test.AssertionsKt PluginClassLoader(plugin=PluginDescriptor(name=Ember Experimental.js, id=com.emberjs.experimental, descriptorPath=plugin.xml, path=~/Library/Application Support/JetBrains/WebStorm2021.3/plugins/EmberExperimental.js, version=2021.3.0, package=null, isBundled=false), packagePrefix=null, instanceId=84, state=active)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:235)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 29 more
```

### Steps to Reproduce

- Install the plugin
- Restart the IDE
- Open an `.hbs` file

**Expected behavior:**

- Plugin doesn't crash

**Actual behavior:**

- Plugin crashes

**Reproduces how often:**

Every time.

### Versions

- The first two lines of the `Help` -> `About` output:

WebStorm 2021.3.1
Build #WS-213.6461.79, built on December 28, 2021

- The version of `intellij-emberjs` (`Settings` -> `Plugins` -> `Ember.js`):

Cannot have this plugin installed because it creates a "Duplicate" error on IDE startup.

**Comment by patricklx** [comment]
I look into it, does the normal ember.js plugin work for you?

**Comment by ghost** [comment]
Thanks :) the normal ember.js plugin does work.

**Comment by patricklx** [comment]
What do you mean with duplicate error? You will need to remove the ember plugin first. Then install this one.
It will take some time until i can test directly on a computer...
So, for now i just tried to update the dependency like in the normal plugin:
https://github.com/patricklx/intellij-emberjs-experimental/actions/runs/1719168516
Maybe you can try it out...

**Comment by ghost** [comment]
Yeah that's what I was saying, there is no choice but to uninstall the ember plugin.

Not sure how would I try it out without a release? Sorry, bit of a plugin noob here 😬

**Comment by patricklx** [comment]
Okay, but if you remove the ember plugin and keep the ember-experimental one you would get the class not found error?
From the link https://github.com/patricklx/intellij-emberjs-experimental/actions/runs/1719168516 is a downloadable plugin at the bottom under artifacts. You can download and then install that in intellij. There is an option to install a plugin from local disk.

**Comment by ghost** [comment]
Same error 🙁 I tried installing IntelliJ and using the plugin (not the local one) there and it works, so I guess that means it's a problem with WebStorm compatibility.

**Comment by patricklx** [comment]
should be fixed in [v2021.3.10](https://github.com/patricklx/intellij-emberjs-experimental/tree/v2021.3.10)
and [v2022.1](https://github.com/patricklx/intellij-emberjs-experimental/tree/v2022.1).
will be released soon
